TurkishFilter/sections/specific.txt

```diff
@@ -1705,7 +1706,7 @@ paylasaninsanlar.blogspot.com##.sidebar > #HTML2
 paylasaninsanlar.blogspot.com##.sidebar > #HTML4
 bianet.org##.banner_main
 bianet.org##.ad52-container
-thegeyik.com,shallwe.com.tr,paragezegeni.com,akillitelefon.com##.adace-slot-wrapper
+egetelgraf.com,thegeyik.com,shallwe.com.tr,paragezegeni.com,akillitelefon.com##.adace-slot-wrapper
```

Comment:
it will break this site
![image](https://user-images.githubusercontent.com/8361299/88181426-7e276500-cc37-11ea-90a8-1a5e5c96f5ad.png)

Reply:
I don't know why they put this video in ad element.